pgjdbc/src/test/java/org/postgresql/test/jdbc2/DatabaseMetaDataTest.java

```diff
@@ -1568,6 +1568,33 @@ void partitionedTables(BinaryMode binaryMode) throws SQLException {
     }
   }
 
+  @MethodSource("data")
+  @ParameterizedTest(name = "binary = {0}")
+  void ancestorTables(BinaryMode binaryMode) throws SQLException {
+    initDatabaseMetaDataTest(binaryMode);
+    if (TestUtil.haveMinimumServerVersion(con, ServerVersion.v11)) {
+      Statement stmt = null;
+      try {
+        stmt = con.createStatement();
```

Comment:
Us try with resources, please

Reply:
ok

Reply:
Because I need to delete the test table in the finally code block, is it necessary to use `try-with-resource` here?